main.go

```diff
@@ -29,6 +30,7 @@ var opts = struct {
 func init() {
 	pflag.IntVarP(&opts.threads, "procs", "p", runtime.NumCPU(), "number of parallel porgrams")
 	pflag.StringVar(&opts.placeholder, "replace", "{}", "replace this string in the command to run")
+	pflag.StringVar(&opts.workerTimeout, "timeout", "0s", "set maximum runtime per queued job (0s == no limit)")
```

Comment:
Then you can use `pflag.DurationVar` here, so parsing is handled by pflag. https://godoc.org/github.com/spf13/pflag#DurationVar